Sources/SmithyHTTPAuthAPI/IdentityResolverConfiguration.swift

```diff
@@ -5,6 +5,11 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
This was moved to `SmithyIdentityAPI` module in previous PR, but it actually belongs in `SmithyHTTPAuthAPI`. Added comment for future reference.